bundles/org.openhab.binding.volvooncall/src/main/java/org/openhab/binding/volvooncall/internal/dto/PositionData.java

```diff
@@ -28,14 +29,21 @@
 public class PositionData {
     public @Nullable Double longitude;
     public @Nullable Double latitude;
-    public @Nullable ZonedDateTime timestamp;
+    private @Nullable ZonedDateTime timestamp;
     public @Nullable String speed;
     private @Nullable String heading;
 
     public Boolean isHeading() {
         return "true".equalsIgnoreCase(heading);
     }
 
+    public Optional<ZonedDateTime> getTimestamp() {
+        ZonedDateTime timestamp = this.timestamp;
+        if (timestamp != null) {
+            return Optional.of(timestamp);
+        }
+        return Optional.empty();
+    }
```

Comment:
```suggestion
    public Optional<ZonedDateTime> getTimestamp() {
        return Optional.ofNullable(timestamp);
    }
```

Reply:
I think I've suggested this before and it can't be done because then it wants you to change the return type to `Optional<@Nullable ZonedDateTime>`. Really wish we could get a null checker that would allow mixins or at least better support the standard libraries...

Reply:
Yes, @cpmeister is right, I've tried this but does not work.